src/clarity-angular/dropdown/_dropdown.clarity.scss

```diff
@@ -67,162 +67,187 @@ $dropdown-white: clr-getColor(lightest);
+            $adjusted_lineHeight: clr-getTypePropertyValueForDomElement(dropdown_text, line-height);
+            $dropdown-text-color: clr-getTextColor();
 
-                background: transparent;
-                border: 0;
-                color: clr-getTextColor();
-                cursor: pointer;
-                display: block;
-                line-height: $adjusted_lineHeight;
-                margin: 0;
-                padding: 1px $clr_baselineRem_1 0;
-                width: 100%;
+            background: transparent;
+            border: 0;
+            color: $dropdown-text-color;
+            cursor: pointer;
+            display: block;
+            height: $adjusted_lineHeight;
+            line-height: $adjusted_lineHeight;
+            margin: 0;
+            padding: 1px $clr_baselineRem_1 0;
+            width: 100%;
 
-                @extend %dropdown-content;
+            @extend %dropdown-content;
 
-                &:hover,
-                &:focus {
-                    text-decoration: none;
-                    background-color: $gray-light;
-                }
+            &:hover,
+            &:focus {
+                background-color: $gray-light;
+                color: $dropdown-text-color;
+                text-decoration: none;
+            }
 
-                &.active {
-                    background: $clr-selection-color;
-                    color: $clr-black;
-                }
+            &.active {
+                background: $clr-selection-color;
+                color: $clr-black;
+            }
 
-                &:focus {
-                    outline: 0;
-                }
+            &:active {
+                box-shadow: none;
+            }
 
-                &.disabled {
-                    cursor: not-allowed;
-                    opacity: 0.4;
-                    user-select: none;
-                }
+            &:focus {
+                outline: 0;
+            }
 
-                &.disabled:hover {
-                    background: none;
-                }
+            &.disabled {
+                cursor: not-allowed;
+                opacity: 0.4;
+                user-select: none;
+            }
 
-                &.disabled:active,
-                &.disabled:focus {
-                    background: none;
-                    box-shadow: none;
-                }
+            &.disabled:hover {
+                background: none;
             }
 
-            .dropdown-divider {
-                border-bottom: 1px solid clr-getColor(light);
-                margin: $clr_baselineRem_0_75 0;
+            &.disabled:active,
+            &.disabled:focus {
+                background: none;
+                box-shadow: none;
             }
         }
 
-        &.open .dropdown-menu {
+        .dropdown-divider {
+            border-bottom: 1px solid clr-getColor(light);
+            margin: $clr_baselineRem_0_75 0;
+        }
+    }
+
+    //Directions
+    .btn-group-overflow,
```

Comment:
I understand why we are doing it this way. But we may want to rethink this approach because it inherently doubles the weight of this block in the generated CSS.

What used to be one block of CSS is now two blocks.